### PR TITLE
Added FAQ on using existing Batch Schema

### LINKED ISSUE #124: Add an FAQ on SBA to SCDF migration

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #124

### CODE REVIEW

File: content/documentation/pages/7-resources/3-faq.md [hunk 1]
```diff
@@ -415,3 +415,12 @@ task launch myBatchApp --arguments="team=cubs"
 However, the preferred way is to write your task or batch application such that it can handle being restarted with a new job instance. One way to do this is to set a `JobParamsIncrementer` for your batch job, as discussed in the Spring Batch [reference guide](https://docs.spring.io/spring-batch/trunk/reference/html/configureJob.html#JobParametersIncrementer).
 
 <!--END_QUESTION-->
+
+<!--QUESTION#useexistingbatchtables-->
+
+Can I use Spring Cloud Data Flow on my existing Spring Batch tables?
```
Comment (sabbyanandan) [inline content/documentation/pages/7-resources/3-faq.md]: What if we rephrase this as: 

> I want to migrate from Spring Batch Admin to Spring Cloud Data Flow. Can I use the existing database that is already used by the Spring Batch jobs?

File: content/documentation/pages/7-resources/3-faq.md [hunk 2]
```diff
@@ -415,3 +415,12 @@ task launch myBatchApp --arguments="team=cubs"
...
+
+No. Spring Cloud Data Flow creates its own schema including the Spring Batch tables.
+The Spring Batch tables created by Spring Cloud Data Flow will be used by your Spring Batch applications and require no change to your apps, they just need to use the same data source configuration as Spring Cloud Data Flow.
```
Comment (sabbyanandan) [inline content/documentation/pages/7-resources/3-faq.md]: The "used by your Spring Batch applications" is a bit unclear. I understand this is for the jobs launched by SCDF, but folks might get confused by referring this to the standalone jobs. Perhaps we could call out the explicit connect with SCDF.

File: content/documentation/pages/7-resources/3-faq.md [hunk 3]
```diff
@@ -427,3 +427,12 @@ There are 3 reasons that this may occur:
 3. You are running a Spring Cloud Task application where the context is held open (for example: if you are using a TaskExecutor). In these cases you can set the `spring.cloud.task.closecontext_enabled` property to `true` when launching your task. This will close the application's context once the task is complete. Thus allowing the application to terminate and record the end time.
 
 <!--END_QUESTION-->
+
+<!--QUESTION#useexistingbatchtables-->
+
+I want to migrate from Spring Batch Admin to Spring Cloud Data Flow. Can I use the existing database that is already used by the Spring Batch jobs?
+
+No. Spring Cloud Data Flow creates its own schema including the Spring Batch tables.
+To allow Spring Cloud Data Flow to show the status of Spring Batch Job executions via the dashboard or shell, your Spring Batch Apps need to use the same data source configuration as Spring Cloud Data Flow.
```
Comment (sabbyanandan) [inline content/documentation/pages/7-resources/3-faq.md]: Perhaps `"datasource"` as a single word and with quotes.

**Comment by cppwfs** [comment]
Rebased, Squashed, Merged
